docs: add better instructions for OTel example (#1896)
keptn/lifecycle-toolkit: docs/content/en/docs/implementing/otel.md

use the following commands.
Use similar commands if you define a different configuration::

> ⚠️ Make sure you have cloned the `lifecycle-toolkit` repository and have `cd` into the correct directory before running the below commands.

Reviewer: ```suggestion
> **Note**
Make sure you have cloned the `lifecycle-toolkit` repository and have `cd`ed into the correct directory (`examples/support/observability`) before running the below commands.
```

use the following commands.
Use similar commands if you define a different configuration::

> **Note**

> Make sure you have cloned the `lifecycle-toolkit` repository and have `cd`ed into the correct directory (`examples/support/observability`) before running the below commands.

Reviewer 2: ```suggestion
You must clone  the `lifecycle-toolkit` repository and `cd` into the correct director
 (`examples/support/observability`) before running the following commands.
```
